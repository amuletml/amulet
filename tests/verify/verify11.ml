let f x = g x
and g = f
