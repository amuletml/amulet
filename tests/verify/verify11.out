verify11.ml[2:5 ..2:5]: error
  Invalid recursive right-hand side for variable g
  • Note: because evaluation of the variable f is not delayed
  Arising from use of the binding
  │ 
2 │ and g = f
  │ 
